src/wasm/kotlin/java/arcs/Addressable.kt

```diff
@@ -0,0 +1,30 @@
+package arcs
+
+/**
+ * Any object implementing this interface can be converted into a stable identifier. This
+ * could a platform specific pointer, identity hashcode, or id as long as it is guaranteed unique.
+ */
+interface Addressable
+
+typealias Address = Int
+
+internal val address2Addressable = kotlin.collections.mutableMapOf<Address, Addressable>()
+internal val addressable2Address = kotlin.collections.mutableMapOf<Addressable, Address>()
```

Comment:
I wonder if this would be nicer as an `object` singleton?

Reply:
I think so, but maybe my Java instincts are kicking in, and Kotlin idioms allow naked globals? :) Jason any comment?